Kernel/Bus/USB/USBConfiguration.h

```diff
@@ -39,7 +47,7 @@ class USBConfiguration {
     ErrorOr<void> enumerate_interfaces();
 
 private:
-    Device& m_device;                              // Reference to the device linked to this configuration
+    Device* m_device;                              // Reference to the device linked to this configuration
```

Comment:
nit: Well Technically Now It's A Pointer, so the comment is invalid-ish

Reply:
I would want to add a nonnull attribute to that, but that seems to be for function arguments/returns only?
Ideally we'd have a transparent nonnullptr/reference wrapper that allows pointer assignment